### ISSUE TITLE
Gluster volume info showing wrong size in gd2 pod

### ISSUE THREAD

**Issue by ksandha** [gluster/glusterd2#1342]
1) create a pvc 
```
[vagrant@kube1 ~]$ kubectl get pvc
NAME       STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
gcs-pvc1   Bound    pvc-624c4d1b-f16c-11e8-bbd1-52540066b71b   2Gi        RWX            glusterfs-csi   8s
[vagrant@kube1 ~]$ 
[vagrant@kube1 ~]$ 

```
2) login to gd2 and check the volume info.

```
[root@gluster-kube1-0 /]# glustercli volume info

Volume Name: pvc-624c4d1b-f16c-11e8-bbd1-52540066b71b
Type: Replicate
Volume ID: 8ba523e7-0a85-4b09-a879-67aa7699342d
State: Started
Capactiy:  2.0 PB
Transport-type: tcp
Options:
Number of Bricks: 3
Brick1: gluster-kube2-0.glusterd2.gcs:/var/run/glusterd2/bricks/pvc-624c4d1b-f16c-11e8-bbd1-52540066b71b/subvol1/brick1/brick
Brick2: gluster-kube1-0.glusterd2.gcs:/var/run/glusterd2/bricks/pvc-624c4d1b-f16c-11e8-bbd1-52540066b71b/subvol1/brick2/brick
Brick3: gluster-kube3-0.glusterd2.gcs:/var/run/glusterd2/bricks/pvc-624c4d1b-f16c-11e8-bbd1-52540066b71b/subvol1/brick3/brick
[root@gluster-kube1-0 /]# 

```

3) The volume info showing wrong volume size. created 2 GB and showing 2 PB.

**Comment by Madhu-1** [comment]
I think this is fixed by #1340
 @aravindavk  please confirm

**Comment by aravindavk** [comment]
Yes, Fixed by #1340

**Comment by aravindavk** [comment]
Closing this issue since issue is fixed by PR #1340, please re-open if not fixed
